springboard_iogt/utils.py

```diff
@@ -100,6 +101,18 @@ def known(cls, indexes, localizer=None):
                 if cls.exists(section.get('name'), indexes)]
 
     @classmethod
+    def get_index(cls, name, indexes):
+        [index] = [index for index in indexes if name in index]
```

Comment:
No, I was going fix this -  I worked on this before getting on the plane - it needs to handle errors better here. If the index is missing, you're going to have problems.